src/config/routes/ErrorBoundary.tsx

```diff
@@ -0,0 +1,71 @@
+import { DateTime } from 'luxon'
+import { useEffect } from 'react'
+import { useRouteError } from 'react-router-dom'
+
+import { useServiceWorkerUpdate } from '../../context'
+import { NotFoundPage } from '../../pages/fallback'
+
+const FAILED_FETCH_ERROR = [
+  'Failed to fetch dynamically imported module',
+  'Importing a module script failed',
+  'error loading dynamically imported module',
+  'dynamically imported module',
+]
+const CHUNK_LOAD_ERROR = 'ChunkLoadError'
```

Comment:
You defined those two in the `helpers` as well, is that intended?

Reply:
I planned on deleting it. Nice catch.